src/providers.rs

```diff
@@ -460,6 +460,15 @@ pub fn do_manage_provider(args: ManageProviderArgs) {
         let mut providers = providers.borrow_mut();
         match providers.get(&args.provider_id) {
             Some(mut provider) => {
+                if let Some(chain_id) = args.chain_id {
+                    log!(
+                        INFO,
+                        "Changing provider {:?} to use chain id: {}",
```

Comment:
Technically, it's not necessarily a change, right? The chain ID might be set to the same chain ID as before (for whatever reason).